test/web3c.js

```diff
@@ -129,13 +129,15 @@ describe('Web3', () => {
     } catch (e) {
       assert.fail(e);
     }
-    await instance.methods.incrementCounter().send({
+    const receipt = await instance.methods.incrementCounter().send({
       from: address,
       gasPrice: '0x3b9aca00',
       gasLimit: '0x100000',
     });
-
-    let logs = await instance.getPastEvents();
```

Comment:
This will get the events from the last produced block, creating a race condition when fetching the logs, since a block can be produced immediately after the block generating the transaction.

Reply:
that's weird semantics. i thought it would get all past events associated with the instance. this seems clearer though.